doc-requirements.txt

```diff
@@ -0,0 +1,6 @@
+# This can not be installed from pip, on Fedora you need to install rpm-python{,3}
+#rpm-python
```

Comment:
What about to remove things which are not needed. I would prefer to remove it completely.